packages/core/src/lib/modal.svelte

```diff
@@ -85,9 +85,10 @@ $: if (typeof document !== 'undefined') {
     <div
       use:clickOutside={handleCloseModal}
       class={cx(
-        'relative max-w-lg border border-medium bg-white p-6 shadow-sm',
+        'relative border border-medium bg-white p-6 shadow-sm',
         {
-          'max-h-[600px] min-h-[400px] w-[480px]': variant === '',
+          'h-[400px] w-[480px]': variant === '',
+          'h-[600px] w-[800px]': variant === 'medium',
```

Comment:
can we do `w-full h-full max-h-[600px] max-w-[800px]` instead? If that doesn't break things it'd be more responsive.

Reply:
Actually these should just be setting width not max width; fixing